packages/chain-mon/.env.example

```diff
@@ -1,3 +1,13 @@
+###############################################################################
+#                               ↓ balance-mon ↓                               #
+###############################################################################
+
+# RPC pointing to network to monitor balances on
+BALANCE_MON__RPC=
+
+# Comma-separated list of addresses to monitor
+BALANCE_MON__ADDRESSES=
```

Comment:
I'm not sure the best way to do this in the `BaseServiceV2` arch, but I think it'd be preferable if we could put names on the addresses like with the previous balance monitor. That would make it much easier to configure dashboards and alerting on multiple network.

https://github.com/ethereum-optimism/optimism/blob/ef5c1f7737cc272a1eb8d880de3bfe1563da9f95/packages/balance-monitor/.env.example#L6-L12

Reply:
Easiest options here are comma separated or JSON array, either works

Reply:
JSON array is slightly easier to read imo

Reply:
Easy to configure:

```
BALANCE_MON__ACCOUNTS='[
  {
    "address": "0x68108902De3A5031197a6eB3b74b3b033e8E8e4d",
    "nickname": "wallet1"
  }
]'
```